backend/tests/endpoints/conftest.py

```diff
@@ -79,3 +79,47 @@ def fixture_mock_dataStorage_delete(mocker):
     zc_core_update_data = AsyncMock()
     mocker.patch("utils.db.DataStorage.delete", side_effect=zc_core_update_data)
     return zc_core_update_data
+
+
+@pytest.fixture(name="init_fake_room")
```

Comment:
I will suggest we create a separate file which will contains all the fake data, this will also reduce the number of arguments passed into the test functions.

Reply:
Yes, please if you can it will be better. I created this fixture because I found out some tests modify the fake data declared at the beginning of the file